python/lsst/ts/Dome/dome_csc.py

```diff
@@ -207,8 +212,7 @@ async def do_moveEl(self, data):
             Contains the data as defined in the SAL XML file.
         """
         self.assert_enabled()
-        # TODO Make sure that radians are used because that is what the real LLCs will use as well. DM-24789
-        cmd = {"moveEl": {"elevation": data.elevation}}
+        cmd = {"moveEl": {"elevation": data.elevation * _DEGREES_TO_RADIANS}}
```

Comment:
so this will be numpy (or np if you use standard import numpy as np) .radians(data.elevation)..etc..

Reply:
I resolved to using meth.radians and math.degrees in almost all cases, except where a list was involved where I am using numpy instead.